### PR TITLE
Add redirects to sites with www subdomains.

### PR DESCRIPTION
**Pull request by richardbporter**

### CODE REVIEW

File: .sites.yml [hunk 1]
```diff
@@ -389,6 +421,8 @@ sites:
         site: uiowa703
         git: 'git@github.com:ITS-UofIowa/ovproperations.research.uiowa.edu.git'
         profile: sitenow
+        redirects:
+            - www.ovproperations.research.uiowa.edu
```
Comment (brandonneil) [inline .sites.yml]: The client "ok'd" us removing the www redirect for the move to acquia cloud. So it is not required.

File: .sites.yml [hunk 2]
```diff
@@ -409,21 +443,29 @@ sites:
         site: uiowa703
         git: 'git@github.com:ITS-UofIowa/rcouncil.research.uiowa.edu.git'
         profile: sitenow
+        redirects:
+            - www.rcouncil.research.uiowa.edu
```
Comment (brandonneil) [inline .sites.yml]: The client "ok'd" us removing the www redirect for the move to acquia cloud. So it is not required.

File: .sites.yml [hunk 3]
```diff
@@ -69,16 +71,22 @@ sites:
...
         profile: sitenow
+        redirects:
+            - www.cgrer.uiowa.edu
     cio.uiowa.edu:
         type: custom
         site: uiowa703
         git: 'git@github.com:ITS-UofIowa/cio.uiowa.edu.git'
         profile: sitenow
+        redirects:
+            - www.cio.uiowa.edu
```
Comment (brandonneil) [inline .sites.yml]: Currently not a domain in AC.

File: .sites.yml [hunk 4]
```diff
@@ -69,16 +71,22 @@ sites:
...
     classrooms.uiowa.edu:
         type: custom
         site: uiowa703
         profile: sitenow
         git: 'git@github.com:ITS-UofIowa/classrooms.uiowa.edu.git'
+        redirects:
+            - www.classrooms.uiowa.edu
```
Comment (brandonneil) [inline .sites.yml]: Currently not a domain in AC.

File: .sites.yml [hunk 5]
```diff
@@ -234,6 +256,8 @@ sites:
         site: uiowa703
         git: 'git@github.com:ITS-UofIowa/hso.research.uiowa.edu.git'
         profile: sitenow
+        redirects:
+            - www.hso.research.uiowa.edu
```
Comment (brandonneil) [inline .sites.yml]: The client dropped this in the move to AC.

File: .sites.yml [hunk 6]
```diff
@@ -314,6 +342,8 @@ sites:
         site: uiowa703
         git: 'git@github.com:ITS-UofIowa/magazine.education.uiowa.edu.git'
         profile: sitenow
+        redirects:
+            - www.magazine.education.uiowa.edu
```
Comment (brandonneil) [inline .sites.yml]: not currently a domain in AC.

File: .sites.yml [hunk 7]
```diff
@@ -559,6 +603,8 @@ sites:
         site: uiowa701
         profile: sitenow_standard_profile
         git: null
+        redirects:
+            - www.cheec.uiowa.edu
```
Comment (brandonneil) [inline .sites.yml]: Not currently a domain in AC.

File: .sites.yml [hunk 8]
```diff
@@ -579,6 +625,8 @@ sites:
         site: uiowa701
         profile: sitenow_standard_profile
         git: null
+        redirects:
+            - www.interprofessional.uiowa.edu
```
Comment (brandonneil) [inline .sites.yml]: Not currently a domain in AC.

**Comment by richardbporter** [comment]
All of those besides www.classrooms.uiowa.edu point to Acquia. They probably should be added as domains or DNS records need to be removed.

File: .sites.yml (hunk 7 above)
Comment (richardbporter) [inline .sites.yml]: Added.

File: .sites.yml (hunk 8 above)
Comment (richardbporter) [inline .sites.yml]: Added.

File: .sites.yml (hunk 4 above)
Comment (richardbporter) [inline .sites.yml]: Added.

File: .sites.yml (hunk 3 above)
Comment (richardbporter) [inline .sites.yml]: Added.

**Pull request merged**